### ISSUE TITLE
Eagle based schematic and PCB available

### ISSUE THREAD

**Issue by abzman** [sjgray/TIB-001-Cart#1]
I found an Eagle based schematic and PCB someone made here: http://news.ide64.org/2018/11/floppy-drive-tib-plc-dd-001-drive-2001.html

I prefer kicad, so I could fork this repo, port over any updates from that Eagle based one, and issue a pull request if you like.

**Comment by sjgray** [comment]
Yes, I have seen that. My project got lost in the pile, so sure go ahead.
Steve

    On Monday, February 14, 2022, 07:46:55 p.m. EST, Evan Allen ***@***.***> wrote:  
 
 


I found an Eagle based schematic and PCB someone made here: http://news.ide64.org/2018/11/floppy-drive-tib-plc-dd-001-drive-2001.html

I prefer kicad, so I could fork this repo, port over any updates from that Eagle based one, and issue a pull request if you like.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
